### ISSUE TITLE
No fan controls after W11 update?

### ISSUE THREAD

**Issue by Tinchox17** [Rem0o/FanControl.Releases#712]
So everything was working fine, i update my windows 11 dev build, and now only the GPU is working.
i try to download and update multiple times, re checked all fans.
but i get "Fan Control A Paired RPM sensor is required" nothing change on my computer.

![image](https://user-images.githubusercontent.com/24707397/166076377-50d9b686-a4ae-4d6d-940f-a7a780d52879.png)

**Comment by twanmal** [comment]
same there, my configuration disappeared completely

**Comment by Tinchox17** [comment]
And if u de add everything again, is unresponsive. If u set to manually 100% or 0 nothing happen.

**Comment by Rem0o** [comment]
Asus mobo? An update seems to have caused the issue. Some users reported than an other update from Armory crate fixed the issue.

**Comment by Tinchox17** [comment]
Yes Asus prime z590, armory I always see the pop up, but I don't think I have installed.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by twanmal** [comment]
which KB please ?

**Comment by Tinchox17** [comment]
i think it was KB5011831, the build was on the last days of may.

Right now, on build 25115.1000 seems to works, but it wont auto start with windows, even if everything is correct.
